functions/chat.ts

```diff
@@ -0,0 +1,77 @@
+import { type PagesFunction } from '@cloudflare/workers-types';
+import { Env } from './lib/env.js';
+import { getKV } from './lib/getKV.js';
+import { GoogleGenAI } from '@google/genai'; // Correct package
+
+// Initialize Google GenAI
+const GEMINI_API_KEY = process.env.GEMINI_API_KEY;
```

Comment:
```suggestion
const { GEMINI_API_KEY } = process.env;
```

Just for less code - these produce the same result - accessing the GEMINI_API_KEY environment variable

Reply:
Done

Reply:
This is not done

`const GEMINI_API_KEY = process.env.GEMINI_API_KEY;`

but I wanted to see

`const { GEMINI_API_KEY } = process.env;`